docs/conf/prague/2018/news/index.rst

```diff
@@ -0,0 +1,13 @@
+:template: {{year}}/{{templatecode}}/generic.html
+
+News
+====
+
+Updates from the team.
+More here as it happens!
+
+.. postlist:: 3
+   :date: %A, %B %d, %Y
+   :format: {title} - {date}
+   :list-style: none
+   :tags: prague-2018
```

Comment:
currently, list is showing community update posts (even though they're not tagged with `prague-2018`)